backend/app/services/document_generation.py

```
@@ -0,0 +1,582 @@
+"""
+Document generation service.
+Orchestrates the four-stage document generation workflow with agent coordination.
+"""
+
+import asyncio
+import json
+from typing import Dict, Any, List, Optional
+from uuid import UUID, uuid4
+from datetime import datetime, timezone
+
+from app.models.document_version import DocumentVersion, DocumentType, DocumentType
```

Ruff (0.13.1)

Redefinition of unused `DocumentType` from line 12

Remove definition: `DocumentType`

(F811)